sdk/docs/pages/typescript-sdk/connect.mdx

```diff
@@ -0,0 +1,53 @@
+# Connecting to Sui Network
+
+The Sui TypeScript SDK provides a SuiClient class that you can use to connect to a network's JSON-RPC server. Use the `SuiClient` for all JSON-RPC read-only operations. 
+
+To establish a connection to a network, import `SuiClient` from `@mysten/sui.js/client` and pass the relevant URL to the `endpoint` parameter. The following example establishes a connection to Testnet and requests SUI from that network's faucet.
+
+```typescript
+import { SuiClient } from '@mysten/sui.js/client';
+// assign Testnet RPC address
+const testnet = 'https://fullnode.testnet.sui.io:443/';
+// set endpoint value to Testnet RPC URL
+const SuiClient = new SuiClient({ endpoint: testnet });
+// get tokens from the Testnet faucet server
+await suiClient.getCoins({
+	owner: '<OWNER_ADDRESS>',
+});
+```
+
+| Connection object | Full node address | `faucet` location |
+| --- | --- | --- |
+| localConnection | `http://127.0.0.1:9000` | `http://127.0.0.1:9123/gas` |
+| devnetConnection | `https://fullnode.devnet.sui.io:443` | `https://faucet.devnet.sui.io/gas` |
+| testnetConnection | `https://fullnode.testnet.sui.io:443` | `https://faucet.testnet.sui.io/gas` |
+| mainnetConnection | `https://fullnode.mainnet.sui.io:443` | `null` |
+
+For local development, you can run `cargo run --bin sui-test-validator` to spin up a local network with a local validator, a Full node, and a faucet server. Refer to [the Local Network guide](https://docs.sui.io/build/sui-local-network) for more information.
+
+```typescript
+import { JsonRpcProvider, localnetConnection } from '@mysten/sui.js';
+// connect to local RPC server
+const provider = new JsonRpcProvider(localnetConnection);
```

Comment:
JsonRpcProvider will be deprecated and replaced with SuiClient (which does not support faucet).

The updated way to request sui from a faucet is documented here: https://github.com/MystenLabs/sui/blob/main/sdk/docs/pages/typescript-sdk/faucet.mdx